cylc/flow/task_pool.py

```diff
@@ -1755,6 +1836,22 @@ def force_trigger_tasks(
 
         return len(unmatched)
 
+    def spawn_parentless_sequential_xtriggers(self):
+        """Spawn successor(s) of parentless wall clock satisfied tasks."""
+        while self.xtrigger_mgr.sequential_spawn_next:
+            taskid = self.xtrigger_mgr.sequential_spawn_next.pop()
```

Comment:
This is a set, so the order of spawning will be arbitrary. Does that matter?

Reply:
I don't think so.. Unless we need predictability .. It would also happen quite fast, anyways.